### PR TITLE
add shrug help

### PR DESCRIPTION
**Pull request by krzyzacy**
ref #8253

/area prow
/assign @stevekuznetsov

### CODE REVIEW

**Comment by BenTheElder** [comment]
I thought we were leaving this undocumented intentionally? :^)

**Comment by krzyzacy** [comment]
eh, I'll update :-))))))

**Comment by krzyzacy** [comment]
@BenTheElder updated :-)

I mean it should not spill out errors..

File: prow/plugins/shrug/shrug.go [hunk 1]
```diff
@@ -47,7 +48,22 @@ type event struct {
...
+func helpProvider(config *plugins.Configuration, enabledRepos []string) (*pluginhelp.PluginHelp, error) {
+	// The Config field is omitted because this plugin is not configurable.
+	pluginHelp := &pluginhelp.PluginHelp{
+		Description: shrugLabel,
+	}
+	pluginHelp.AddCommand(pluginhelp.Command{
+		Usage:       "/[un]shrug",
+		Description: shrugLabel,
+		Featured:    false,
+		WhoCanUse:   "Anyone, " + shrugLabel,
```
Comment (cjwagner) [inline prow/plugins/shrug/shrug.go]: 😄

**Pull request merged**